timber/src/test/java/timber/log/TimberTest.java

```diff
@@ -384,6 +457,17 @@ private static void assertExceptionLogged(String message, String exceptionClassn
     assertThat(log.throwable).isNull();
   }
 
+  private static void assertExceptionLogged(int logType, String exceptionClassname, String tag) {
```

Comment:
Just have it call this method with a `null` tag and then change the tag assertion to
```java
assertThat(log.tag).isEqualTo(tag != null ? tag : "TimberTest");
```